src/custom/hooks/useSwapCallback.ts

```diff
@@ -106,7 +110,7 @@ export function useSwapCallback(
           kind,
           account,
           chainId,
-          inputAmount,
```

Comment:
but if we feed UNI with the amount+fees they will think we trade more than we trade ---> they will give us a price for the wrong amount